Makefile

```diff
@@ -14,7 +14,16 @@
 
 COMMONENVVAR=GOOS=$(shell uname -s | tr A-Z a-z) GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m)))
 BUILDENVVAR=CGO_ENABLED=0
-REG_PORT=5000
+
+LOCAL_REGISTRY=localhost:5000/scheduler-plugins
+LOCAL_IMAGE=kube-scheduler:latest
+
+# RELEASE_REGISTRY is the container registry to push
+# into. The default is to push to the staging
+# registry, not production(k8s.gcr.io).
+RELEASE_REGISTRY?=gcr.io/k8s-staging-scheduler-plugins
+RELEASE_VERSION?=$(shell git describe --tags --match "v*")
```

Comment:
This line of code determines the container image tags that are created for the staging registry. It does not determine the container image tags for the production registry(k8s.gcr.io). The tags for the production registry are set at image promotion time using the below process.

https://github.com/kubernetes/k8s.io/tree/master/k8s.gcr.io#image-promoter

Let me know if you want this command changed to something else. It should be something that is meaningful to the scheduler-plugin maintainers, so that they can easily select the correct image to promote when following the image promoter pull request process.

Reply:
Ah, that's nice. Thanks for the explanation.